pandas/formats/format.py

```diff
@@ -906,8 +921,50 @@ def get_col_type(dtype):
                          if x else '{}') for x in row]
             else:
                 crow = [x if x else '{}' for x in row]
+            if i < clevels and self.fmt.header and self.multicol:
+                row2 = list(crow[:ilevels])
+                ncol = 1
+                coltext = ''
+
+                def appendCol():
```

Comment:
it's an internal function, so not that important, but let's be consistent in PEP 8 naming, so appendCol -> append_col